src/components/Home.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react'
+import { Link } from 'react-router-dom'
+import '../styles/home.css';
+
+class Home extends Component {
+
+    constructor() {
+        super()
+        this.state = {
+            users: {
+                user1: { backgroundColor: 'Blue' },
+                user2: { backgroundColor: 'Green' },
+                user3: { backgroundColor: 'purple' },
+                user4: { backgroundColor: 'Red' },
+            },
+        }
+    }
+
+    render() {
+        return (
+                <div id="homeComponent">
+                    <header>WHO'S WATCHING?</header>
+                    <div id='users'>
+                        <Link to='/Catalog' >
+                            <div className='user' style={this.state.users.user1}><p>user1</p></div>
```

Comment:
use map instead. What happens if you want to add more users? right now it is hard coded

save the users in an array in state and then map over it rendering the user div. 

```
this.state.users.map(user => {
return(
<div>user.name</div>
)})
```